packages/block-editor/src/hooks/duotone.js

```diff
@@ -68,7 +68,7 @@ export function getValuesFromColors( colors = [] ) {
 function DuotoneFilter( { selector, id, values } ) {
 	const stylesheet = `
 ${ selector } {
-	filter: url( #${ id } );
+	filter: url( #${ id } ) !important; // We need !important to overide rules that come from theme.json.
```

Comment:
mmm, the user-provided duotone ends up with a selector such as `.wp-duotone-filter-UUID img` (011) and the global one is `.wp-block-image img` (011). Isn't it enough that the user-provided duotone is enqueued after theme.json one to win?

Reply:
The one for the frontend we can remove yeah but this one is used in the editor. The problem is that in the editor the theme.json rules have `.editor-styles-wrapper` prepended to them, which means they have a higher specificity so they the precedence over the user provided rules.